plugins/lsp_java.lua

```diff
@@ -0,0 +1,47 @@
+-- mod-version:3 -- lite-xl 2.1
+
+local common = require "core.common"
+local config = require "core.config"
+local lsp = require "plugins.lsp"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_java"
+local installed_path_library = USERDIR .. PATHSEP .. "libraries" .. PATHSEP .. "jdk"
+
+local platform
+if PLATFORM == "Windows" then
+  platform = "win"
+elseif PLATFORM == "Mac OS X" then
+  platform = "mac"
+else
+  platform = "linux"
+end
+
+local jdtls_data_path = system.absolute_path(".") .. PATHSEP .. ".jdtls"
+system.mkdir(jdtls_data_path)
```

Comment:
Does it need to be in the project dir?

Reply:
From the official REAMDE: `This should be unique per workspace/project.`